code/modules/mob/living/simple_animal/hostile/megafauna/chaos_marine.dm

```diff
@@ -0,0 +1,510 @@
+		new /mob/living/simple_animal/hostile/chaos/greater(X2)
+	else
+		new /mob/living/simple_animal/hostile/chaos(X1)
+		new /mob/living/simple_animal/hostile/chaos(X2)
+	SLEEP_CHECK_DEATH(20 * dash_mod)
+	remove_atom_colour(TEMPORARY_COLOUR_PRIORITY, newcolor)
+	cmdepower()
+	infernal_summon_cooldown = (world.time + initial(infernal_summon_cooldown))
+	charging = FALSE
+
+/mob/living/simple_animal/hostile/megafauna/chaos_marine/proc/shoot_projectile(turf/marker, set_angle)
+	if(!isnum(set_angle) && (!marker || marker == loc))
+		return
+	var/turf/startloc = get_turf(src)
+	var/obj/projectile/P = new /obj/projectile/bullet/chaos_bomb/blood(startloc)
+	P.preparePixelProjectile(marker, startloc)
+	P.firer = src
+	if(target)
+		P.original = target
+	P.fire(set_angle)
+
+/mob/living/simple_animal/hostile/megafauna/chaos_marine/proc/telegraph()
+	for(var/mob/M in range(10,src))
+		if(M.client)
+			flash_color(M.client, "#FF0000", 2)
+			shake_camera(M, 2, 1)
+	playsound(src, 'sound/magic/clockwork/narsie_attack.ogg', 200, TRUE)
+
+/mob/living/simple_animal/hostile/megafauna/chaos_marine/proc/cmempower()
+	damage_coeff = list(BRUTE = 0.6, BURN = 0.2, TOX = 0.2, CLONE = 0.2, STAMINA = 0, OXY = 0.2)
+	add_atom_colour(newcolor, TEMPORARY_COLOUR_PRIORITY)
+	new /obj/effect/temp_visual/cult/sparks(get_turf(src))
+
+/mob/living/simple_animal/hostile/megafauna/chaos_marine/proc/cmdepower()
+	damage_coeff = initial(damage_coeff)
+	remove_atom_colour(TEMPORARY_COLOUR_PRIORITY, newcolor)
+	new /obj/effect/temp_visual/cult/sparks(get_turf(src))
+
+/obj/item/clothing/suit/space/hardsuit/ert/paranormal/berserker/chaos
+	name = "berserker's hardsuit"
+	allowed = list(/obj/item/gun, /obj/item/nullrod, /obj/item/tank/internals)
+	armor = list("melee" = 75, "bullet" = 50, "laser" = 30, "energy" = 30, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	resistance_flags = FIRE_PROOF | LAVA_PROOF
+	cell = /obj/item/stock_parts/cell/high/slime
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/paranormal/berserker/chaos
+	icon_state = "hardsuit-chaos"
+	icon = 'icons/Fulpicons/fulpclothing_worn.dmi'
+	worn_icon = 'icons/Fulpicons/fulpclothing_worn.dmi'
+
+/obj/item/clothing/head/helmet/space/hardsuit/ert/paranormal/berserker/chaos
+	name = "berserker's helmet"
+	armor = list("melee" = 75, "bullet" = 50, "laser" = 30, "energy" = 30, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	resistance_flags = FIRE_PROOF | LAVA_PROOF
+	icon_state = "hardsuit0-chaos"
+	icon = 'icons/Fulpicons/fulpclothing_worn.dmi'
+	worn_icon = 'icons/Fulpicons/fulpclothing_worn.dmi'
+
+/obj/item/clothing/suit/space/hardsuit/ert/paranormal/berserker/chaos/process()
+	. = ..()
+	var/mob/living/carbon/C = loc
+	if(istype(C) && prob(4))
+		if(prob(25))
+			C.adjustOrganLoss(ORGAN_SLOT_BRAIN, 5, 20)
```

Comment:
is it intentional that this hardsuit won't bring your brain damage above 20/brain health below 180?

Reply:
Not intentional, didn't check how it actuall works.